docs/architecture/cloud-native/application-bundles.md

```diff
@@ -1,20 +1,20 @@
 ---
 title: Cloud Native Application Bundles
 description: Architecting Cloud Native .NET Apps for Azure | Cloud Native Application Bundles
-ms.date: 05/13/2020
+ms.date: 01/19/2021
 ---
 
 # Cloud Native Application Bundles
 
 A key property of cloud-native applications is that they leverage the capabilities of the cloud to speed up development. This design often means that a full application uses different kinds of technologies. Applications may be shipped in Docker containers, some services may use Azure Functions, while other parts may run directly on virtual machines allocated on large metal servers with hardware GPU acceleration. No two cloud-native applications are the same, so it's been difficult to provide a single mechanism for shipping them.
 
-The Docker containers may run on Kubernetes using a Helm Chart for deployment. The Azure Functions may be allocated using Terraform templates. Finally, the virtual machines may be allocated using Terraform but built out using Ansible. This is a whole mess of technologies and there has been no way to package them all together into a reasonable package. Until now.
+The Docker containers may run on Kubernetes using a Helm Chart for deployment. The Azure Functions may be allocated using Terraform templates. Finally, the virtual machines may be allocated using Terraform but built out using Ansible. This aspect is a whole mess of technologies and there has been no way to package them all together into a reasonable package. Until now.
```

Comment:
"Whole mess"??  Maybe, a "large variety" might be more appropriate.

Reply:
Thanks @robvet. Yes, that would be better. Included it in the latest commit.